### PR TITLE
BWDB-12072

### PR DESCRIPTION
**Pull request by jhough07**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes

1. Updating the root URL for 10dlc campaigns
2. Adding Brand External Vetting endpoints
3. Adding first and last name to brand payloads

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-12072.s3-website-us-east-1.amazonaws.com/

File: campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md [hunk 1]
```diff
@@ -980,6 +1005,247 @@ HTTP/1.1 429 Too Many Requests
...
+         <VettingToken>YAQ45SM</VettingToken>
+         <VettingScore>0</VettingScore>
+         <VettingClass>STANDARD</VettingClass>
+         <VettingStatus>PENDING</VettingStatus>
+         <CreateDate>2021-09-27T19:09:40.981Z</CreateDate>
+         <VettedDate>2021-09-27T19:09:40.981Z</VettedDate>
+         <BrandId>B111111</BrandId>
+       </BrandVetting>
+    </BrandVettings>
+    <TotalCount>1</TotalCount>
```
Comment (maksymmateiuk) [inline campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md]: API doesn't have TotalCount.

Comment (jhough07) [reply campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md]: good catch

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-12072.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-12072.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-12072.s3-website-us-east-1.amazonaws.com/

**Review by AswiniKumarV — APPROVED** [review]

**Review by maksymmateiuk — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**